src/definitions/modules/dropdown.js

```diff
@@ -3838,24 +3838,21 @@
 
                         return text.replace(regExp.escape, '\\$&');
                     },
-                    htmlEntities: function (string, forceAmpersand) {
-                        forceAmpersand = typeof forceAmpersand === 'number' ? false : forceAmpersand;
+                    htmlEntities: function (string) {
```

Comment:
IDK why `forceAmpersand` was introduced in https://github.com/fomantic/Fomantic-UI/pull/2705. What is the usecase when ampersand "must not be escaped to make things safe"?

Reply:
It was introduced as described in the related PR description because of " a possible XSS through an entity encoded select option text when converted into a FUI dropdown."
Here is the original jsfiddle (adjusted to run FUI 2.9.2 which had the issue back then to be able to reproduce the issue.

Select the obvious evil dropdown entry to see the issue

https://jsfiddle.net/lubber/norj3k7p/1/

Reply:
We probably need to completely rewrite the whole data handling of dropdown in all possible situations (seek through all the dropdown issues the past years to find related jsfiddles...) to clean this up. 
Most difficult part always were
-  comparing/check for existing values, epecially on "mutlitple"
- already entity encoded values, avoid "double encoding" (which was one reason not to encode the ampersand again, otherwise the result of &amp; was &amp;amp;, for data which was meant to be stored, not rendered

Reply:
> XSS issue

I tested it using my PR and XSS is not present.

>  need to completely rewrite the whole data handling of dropdown

I completely agree.

> "double encoding" 

Yes, this is the problem. Not only `&amp;`, but also `&quote;` stripping. This is what this PR addresses and I described it shortly in the PR title/description.